_includes/app/txn-sample.py

```diff
@@ -26,33 +26,33 @@ def txnWrapper(conn, op):
                 onestmt(conn, "RELEASE SAVEPOINT cockroach_restart")
                 break
 
-            except psycopg2.DatabaseError as e:
-                if e.pgcode != '40001':
+            except psycopg2.OperationalError as e:
+                if e.pgcode != psycopg2.errorcodes.SERIALIZATION_FAILURE:
```

Comment:
`40001` will work in all languages, but many languages have more meaningful names available for this error code; we should probably use these names in examples where possible.